generate_life_calendar.py

```diff
@@ -272,11 +277,16 @@ def main():
                          nargs='?', const='today', help='Darken until date. '
                         '(defaults to today if argument is not given)')
 
+    parser.add_argument('--classic', dest='classic_mode',default=False, action='store_true', help='generate the original format from WBW')
     args = parser.parse_args()
+
+    if not ((args.date is not None)  ^ args.classic_mode):
```

Comment:
this seems like a strange use of bitwise XOR... why not just "if (args.date is None) and (args.classic_mode == False)" ?

Reply:
if both of them are ON we need to shout.Your expression does not catch it.

  even better: if bool(args.date) == args.classic_mode: